src/conspiracies/relationextraction/other/align.py

```diff
@@ -0,0 +1,115 @@
+"""Directly copied from the spacy-transformers library https://raw.githubusercontent.com/explosion/spacy-transformers/master/spacy_transformers/align.py
+Pasted here to avoid clashing torch versions
+"""
```

Comment:
```suggestion
"""
Copyright (C) 2022 Explosion AI and Center for Humanities Computing Aarhus - All Rights Reserved
You may use, distribute and modify this code under the
terms of the MIT license.
Original code from:
https://raw.githubusercontent.com/explosion/spacy-transformers/master/spacy_transformers/align.py

The following functions are copied/modified from their code:
[...]
To avoid clashing torch versions.
"""
```

Remove chcaa if we haven't made any changes